index.html

```diff
@@ -1,11 +1,88 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-  <head>
+<head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Homework 1</title>
-  </head>
-  <body>
-    <h1>Start developing here</h1>
-  </body>
+</head>
+<body>
+<header>
+    <h3><a href="https://web-studio"><span>Web</span>Studio</a></h3>
```

Comment:
you don't need an h3 tag here - this is not a page, section or any other title, this is a logo